docs/download_files.md

```diff
@@ -98,6 +98,7 @@ The `My Dataset` button can be used to view and download the custom dataset as a
 <!--TODO: Confirm that these are the correct names that will be used as the folder names for data format-->
 Each zip file will be named with a unique dataset ID, the chosen data format (either `SingleCellExperiment` or `AnnData`), and the date you accessed the data on the ScPCA Portal.
 Note that a custom dataset can only contain data in one format, [`SingleCellExperiment` objects (`.rds` files)](#custom-datasets-with-singlecellexperiment-format) or [`AnnData` objects (`.h5ad` files)](#custom-datasets-with-anndata-format) (see {ref}`Why can't I change the data format in My Dataset <faq:why can't i change the data format in my dataset>`). 
+If a sample has [spatial transcriptomics data](#spatial-transcriptomics-libraries), the `Spatial` box can be used to indicate whether or not to include the spatial transcriptomics data in the download. 
```

Comment:
Suggesting a rephrase so that we can link the modalities section, and use the `My Dataset` wording:

```suggestion
If a sample has [spatial transcriptomics data](#spatial-transcriptomics-libraries), you can check the {ref}`Spatial modality box<download_options:Modalities>` to include the spatial transcriptomics data in `My Dataset`.
```